net-print/cnijfilter/cnijfilter-3.80.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2012 Gentoo Foundation
```

Comment:
Same comments as for the other package, for a start.

Reply:
Fixed